README.md

```diff
@@ -18,18 +18,23 @@ NGSA Java App is inteneded for platform testing and monitoring in one or many Ku
 ```text
 
 Usage:
-        mvn clean spring-boot:run -Dspring-boot.run.arguments=[options] 
+        mvn clean spring-boot:run -Dspring-boot.run.arguments=[options]
+        
+        export [env var]=<value>
+        mvn clean spring-boot:run
```

Comment:
I set the following env vars:
```
 $ env | grep BURST
 BURST_TARGET=50
 BURST_SERVICE=
 BURST_HEADER=true
```

and ran the app using 
`mvn clean spring-boot:run -Dspring-boot.run.arguments="--log-level=info --burst-header=true --burst-target=30"`
This was the burst value: `X-Load-Feedback: service=, current-load=9, target-load=30, max-load=80`. Looks like the CLI args are taking precedence, is this intentional? Not sure if this might be just a spring-ism